backstage/templates/project-create/pull-request-changes/catalog-info.yaml.njk

```diff
@@ -28,3 +28,19 @@ spec:
     {%- for member in values.editors %}
     - ${{member.ref}}
     {%- endfor %}
+
+---
+apiVersion: backstage.io/v1alpha1
+kind: Group
+metadata:
+  name: ${{values.projectId}}-viewers
+  title: ${{values.projectName}} Viewers
+  annotations:
+    cloud.google.com/project: ${{values.projectId}}
+spec:
+  type: team
+  children: []
+  members:
```

Comment:
Does this work in Backstage if there are no members?

We may want to add the same if/else branch as this change from late yesterday? For teh claim, the viewers don't work with the empty/`null` value.

https://github.com/PHACDataHub/sci-portal/pull/437/files